util/envDynawo.sh

```diff
@@ -650,8 +642,8 @@ config_3rd_party() {
   if [ $DYNAWO_LIBARCHIVE_HOME_DEFAULT != true ]; then
     CMAKE_OPTIONAL="$CMAKE_OPTIONAL -DLIBARCHIVE_HOME=$DYNAWO_LIBARCHIVE_HOME"
   fi
-  case $DYNAWO_BUILD_TYPE in
-    Tests|TestCoverage)
+  case $DYNAWO_BUILD_TYPE_THIRD_PARTY in
+    Debug)
```

Comment:
why removing the Tests target?

Reply:
DYNAWO_BUILD_TYPE_THIRD_PARTY is only Release or Debug
DYNAWO_BUILD_TYPE can be Tests and TestCoverage